partner/app/main.py

```diff
@@ -0,0 +1,76 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import sys
+from typing import List
+
+from fastapi import FastAPI, Query
+from pydantic import BaseModel
+
+version = f"{sys.version_info.major}.{sys.version_info.minor}"
+
+app = FastAPI()
+
+
+class Tile(BaseModel):
+    id: int
+    name: str
+    click_url: str
+    image_url: str
+    impression_url: str
+    advertiser_url: str
+
+
+TILES = [
```

Comment:
change request: The contile filter validation system attempts to parse these as valid URLs before checking the host. We may want to use something that is more parseable (e.g. `https://example.com/click_url`) As these are, they would never pass Contile filtering.

Reply:
That makes sense! I'll update that. Thanks!